### DISCUSSION TITLE
Higher quality audio

### DISCUSSION THREAD

**Discussion by composition-cassidy** [neonbjb/tortoise-tts#320] [Q&A]
Is it possible to train it and have it output higher quality audio? (Like 44100 hz or even 48000 hz)

**Comment by RahulBhalley** [comment]
Maybe you can use some audio super resolution model after generating 22kHz audio from Tortoise.

**Comment by fakerybakery** [comment]
Try [VoiceFixer](https://github.com/voicefixer/voicefixer)

**Comment by RahulBhalley** [comment]
Ok.
